bsb_nest/device.py

```diff
@@ -21,6 +21,8 @@ class NestDevice(DeviceModel):
         type=types.or_(Targetting, NestRule), default=dict, call_default=True
     )
     """Targets of the device, which should be either a population or a nest rule"""
+    receptor_type = config.attr(type=int, required=False, default=0)
```

Comment:
Is default 0 backwards compatible? i.e., is it the NEST default?

Reply:
Yes it is.